packages/valory/skills/counter/tests/test_counter.py

```diff
@@ -68,6 +68,14 @@ def setup_class(cls, **kwargs: Any) -> None:
             AbciDialogues, cls._skill.skill_context.abci_dialogues
         )
 
+    @classmethod
```

Comment:
Remove class method; also add `super.setup()`

Reply:
No point in calling `super().setup()` since it basically does what `setup_class` does which is setting up the skill